### ISSUE TITLE
Support for Gitlab

### ISSUE THREAD

**Issue by minecraftchest1** [utterance/utterances#516]
I have found this project, and I love it. It is simple, requires almost no setup, and it just works. However, I think somthing that would make this project better is support for gitlab for people using gitlab pages. 

Perhaps we could make this part of utterance a module that could be easily replaced?

I don't know any typescript or I would look into adding this feature.

**Comment by nschonni** [comment]
Dupe of a few other issues https://github.com/utterance/utterances/search?q=Gitlab&type=issues

**Comment by minecraftchest1** [comment]
I see almost all of them were closed without reason, and the one that did have a reason, was updated with new information and was ignored.

**Comment by smalltimer** [comment]
Yeah, integration with gitlab would be awesome. I have moved most of stuff on there as $M S has now taken over github.

**Comment by kecskemethy** [comment]
from above dupe list #240 has some useful / worth reading info imho. but as I see Jeremy is not really a fun of these ideas see #508

**Comment by Semiotikal** [comment]
bumping this for gitlab support

**Comment by CosteaPaul** [comment]
Any life left in this idea? 
Integration of this with gitlab pages would indeed be pretty sweet.

**Comment by famuvie** [comment]
Annual reminder that this would be greatly appreciated.
